data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -37,35 +37,35 @@
             {% evaluate locationId using 'ID/Location' PL: firstSegments.PV1.3 -%}
             {% if locationId -%}
                 {% include 'Resource/Location' PL: firstSegments.PV1.3, ID: locationId -%}
+                {% include 'Resource/Encounter' PV1: firstSegments.PV1, Location_ID_PV1_3: locationId, ID: encounterId -%}
             {% endif -%}
 
             {% evaluate locationId using 'ID/Location' PL: firstSegments.PV1.6 -%}
             {% if locationId -%}
                 {% include 'Resource/Location' PL: firstSegments.PV1.6, ID: locationId -%}
+                {% include 'Resource/Encounter' PV1: firstSegments.PV1, Location_ID_PV1_6: locationId, ID: encounterId -%}
             {% endif -%}
 
             {% if patientId -%}
                 {% include 'Reference/Encounter/Subject' ID: encounterId, REF: fullPatientId -%}
             {% endif -%}
         {% endif -%}
 
-        {% assign pr1SegmentLists = hl7v2Data | get_segment_lists: 'PR1' -%}
-        {% for pr1Segment in pr1SegmentLists.PR1 -%}
-            {% evaluate procedureId using 'ID/Procedure' PR1: pr1Segment, baseId: patientId -%}
-            {% if procedureId -%}
-                {% include 'Resource/Procedure' PR1: pr1Segment, ID: procedureId -%}
+        {% evaluate procedureId using 'ID/Procedure' PR1: firstSegments.PR1, baseId: patientId -%}
```

Comment:
Why do we change it from segmentList to firstSegment?

Reply:
I believe it's a mistake, I'll fix it.